index.html

```diff
@@ -7,59 +7,216 @@
-				<h1 class="primary">
-					Hey there. I’m Andrés 
-				</h1>
-				<h2 class="primary">
-					I’m a software developer
-				</h2>
-			</div>
-			<p class="supporting-text">
-				 I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
-don’t hestiate to contact me.
-			</p>
-		</section>
-		<section class="social-media">
+			<h1>I’m Andrés Glad to see you!</h1>
+			<p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+			<span>LET’S CONNECT</span>
+			<section class="headline__social-media">
 				<ul class="social-media__ul">
 					<li>
-						<a href="https://twitter.com/andres_condezo" ><img src="./img/twitter_icon.svg" alt="twitter logo"></a>
+						<a href="https://twitter.com/andres_condezo" rel="noreferrer noopener" target="_blank"><img src="./img/twitter_icon.svg" alt="twitter logo"></a>
 					</li>
 					<li>
-						<a href="https://www.linkedin.com/in/andres-condezo/"  ><img src="./img/linkedin_icon.svg" alt="linkedin logo"></a>
+						<a href="https://www.linkedin.com/in/andres-condezo/" rel="noreferrer noopener" target="_blank" ><img src="./img/linkedin_icon.svg" alt="linkedin logo"></a>
 					</li>
 					<li>
-						<a href="https://ra-condezomonge.medium.com"  ><img src="./img/medium_icon.svg" alt="medium logo"></a>
+						<a href="https://ra-condezomonge.medium.com" rel="noreferrer noopener" target="_blank" ><img src="./img/medium_icon.svg" alt="medium logo"></a>
 					</li>
 					<li>
-						<a href="https://github.com/andres-condezo"  ><img src="./img/github_icon.svg" alt="github logo"></a>
+						<a href="https://github.com/andres-condezo" rel="noreferrer noopener" target="_blank" ><img src="./img/github_icon.svg" alt="github logo"></a>
 					</li>
 					<li>
-						<a href="#" ><img src="./img/angellist_icon.svg"  alt="angelist logo"></a>
+						<a href="#" ><img src="./img/like_icon.svg"  alt="like logo"></a>
 					</li>
 				</ul>
+			</section>
+		</section>
+
+		<section class="works">
+
+			<article class="works__card">
+				<figure class="snapshoot"><img src="./img/Snapshoot-Portfolio.png" alt=""></figure>
+				<section class="works__left-block">
+					<div class="works__primary-text">
+						<h3 class="works__project-title">Tonic</h3>
+						<div class="frame-2">
+							<span class="client">CANOPY</span>
+							<div class="counter"></div>
+							<span class="role">Back End Dev</span>
+							<div class="counter"></div>
+							<span class="year">2015</span>
+						</div>
+					</div>
+					<p class="works__primary-par">
+						A daily selection of privately personalized reads; no accounts or sign-ups required.
+					</p>
+					<div class="works__tags">
+						<span>html</span>
+						<span>css</span>
+						<span>javaScript</span>
+					</div>
```

Comment:
- [ ] You've implemented this section extremely well in terms of the Figma design 👍. That's a really great just. I would just like you to use a `ul` tag here for correct semantics. I know it doesn't make any difference visually. but in order to adhere to correct HTML semantics it makes sense to use a `ul` here with `li` children.

something like this would be fine.
```suggestion
					<ul class="works__tags">
						<li><span>html</span></li>
						<li><span>css</span></li>
						<li><span>javaScript</span></li>
					</ul>
```

Reply:
Hi @MrBrN197 , Right now I make the changes.